packages/valory/agents/registration_start_up/aea-config.yaml

```diff
@@ -96,14 +96,14 @@ public_id: valory/p2p_libp2p_client:0.1.0
 type: connection
 config:
   nodes:
-  - uri: ${P2P_URI:str:acn.staging.autonolas.tech:9006}
-    public_key: ${P2P_PUBLIC_KEY:str:02e741c62d706e1dcf6986bf37fa74b98681bc32669623ac9ee6ff72488d4f59e8}
+  - uri: ${P2P_URI:str:localhost:11000}
+    public_key: ${P2P_PUBLIC_KEY:str:03c74dbfbe7bbc1b42429f78778017a3cd7eaf9d59d1634c9505a3f7c1a9350e71}
```

Comment:
if I want to overwrite this via `BaseTestEnd2End.extra_configs`, how does one overwrite a with a list of dict? e.g. I want to do something like this:

```
        {
            "dotted_path": f"{__p2p_prefix}.config.nodes",
            "value": [dict(uri="acn.staging.autonolas.tech:9006",
                           public_key="02e741c62d706e1dcf6986bf37fa74b98681bc32669623ac9ee6ff72488d4f59e8")
                      ],
        },
```

Reply:
Still relevant?

Reply:
no it's not important, may in fact be solved by specifying the json string directly (suggested by @Adamantios)